test/soca/gw/jjob_prep.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+bindir=$1
+srcdir=$2
+
+# export env. var.
+source ${srcdir}/test/soca/gw/runtime_vars.sh $bindir $srcdir
+
+# prepare background from previous cycle
+mkdir -p $bindir/test/soca/gw/testrun/ROTDIRS/gdas_test/gdas.20180415/06/ocean/
```

Comment:
Same comment, don't use hard coded path/dates here

Reply:
done as well. I did also run shellcheck since my bash style was all over the place.